### PR TITLE
Chuckles bugfixes

### PR DESCRIPTION
**Pull request by mike12345567**
Not merging this yet, creating PR to add any bugs that have been fixed to it.

Bugs:

Fix deletion of app data when deleting an app - destroying pouch instance DBs when deleting an app (#716)
Auto screen deletion when table deleted (#700) - this is not fully resolved due to #717 
Fixes prettier svelte plugin breaking on some syntax by upgrading from 0.7.0 to 1.1.0

Updated table delet modal to show what is going to be removed:
image

### CODE REVIEW

File: packages/builder/src/builderStore/store/index.js [hunk 1]
```diff
@@ -185,6 +186,26 @@ const setCurrentScreen = store => screenName => {
...
+    screens = [screens]
+  }
+  store.update(state => {
+    if (pageName == null) {
+      pageName = state.pages.main.name
+    }
+    for (let screen of screens) {
+      state.screens = state.screens.filter(c => c.name !== screen.name)
+      // Remove screen from current page as well
+      state.pages[pageName]._screens = state.pages[pageName]._screens.filter(
```
Comment (aptkingston) [inline packages/builder/src/builderStore/store/index.js:200]: Using `pageName` here might mean that if the user was on the unathenticated page, then went to backend and deleted a screen it wouldn't actually remove it from the "page" structure. I'd replace this with just `state.pages.main._screens` but double check that still works. The `pageName` var is just "main" or "unauthenticated" I think, but the screens we want to delete are always under "main",

**Review by aptkingston — CHANGES_REQUESTED** [review]
LGTM! Just that one wee change.

File: packages/builder/src/builderStore/store/index.js (hunk 1 above)
Comment (mike12345567) [reply packages/builder/src/builderStore/store/index.js:200]: I only use page name here as I used the stuff that was previously part of the actual screen delete pop over, if the `pageName` is null then I do:
```
if (pageName == null) {
      pageName = state.pages.main.name
    }
```

in the case of backend code for deleting a table we don't set the page name so it always uses main instead. For the screen popover it needs to be able to delete pages from the `unauthenticated` page so I had to have the page name in for that.

**Review by aptkingston — APPROVED** [review]

File: packages/builder/src/builderStore/store/index.js (hunk 1 above)
Comment (aptkingston) [reply packages/builder/src/builderStore/store/index.js:200]: No worries, if that's null when it's used then all good.

**Pull request merged**